lib/std/os.zig

```diff
@@ -3360,7 +3360,7 @@ pub const WaitPidResult = struct {
 };
 
 pub fn waitpid(pid: pid_t, flags: u32) WaitPidResult {
-    const Status = if (builtin.link_libc) c_uint else u32;
+    const Status = if (builtin.link_libc) c_int else i32;
```

Comment:
changing this to `i32` is not correct, see `std.os.linux.waitpid`
```suggestion
    const Status = if (builtin.link_libc) c_int else u32;
```

Reply:
i see, fixed